Formula/dvm.rb

```diff
@@ -15,14 +15,16 @@ class Dvm < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["PATH"] = "#{ENV["PATH"]}:#{buildpath}/bin"
```

Comment:
@DomT4 I'm not quite sure what that append_path function is doing. 😊  I tried replacing the entire line with the line above and while it executes without error, the current PATH isn't updated with the right value and the build fails.